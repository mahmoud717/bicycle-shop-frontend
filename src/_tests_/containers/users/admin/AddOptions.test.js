import React from 'react';

import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import AddOptions from '../../../../containers/users/admin/addOptions';

const mockStore = configureMockStore();
const store = mockStore({});
const setup = () => shallow(
  <Provider store={store}>
    <AddOptions />
  </Provider>,
);

describe('add options Component', () => {
  let component;
  beforeEach(() => {
    component = setup();
  });
  test('renders the add options container', () => {
    const page = component.find('.add-options');
    expect(page).toMatchSnapshot();
  });
});
