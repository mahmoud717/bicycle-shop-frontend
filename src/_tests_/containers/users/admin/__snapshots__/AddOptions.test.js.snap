// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add options Component renders the add options container 1`] = `ShallowWrapper {}`;
